fallout/gatling/gatling_timeseries_latency.yaml

```diff
@@ -0,0 +1,99 @@
+# master uses a Gatling DSE plugin based on 3.x
+# DSP-17972-unified-driver uses a Gatling DSE plugin based on the unified driver (>= 4.5.x)
+branch: "master"
+
+---
+
+ensemble:
+  observer:
+    node.count: 1
+    provisioner:
+      name: ctool
+      properties:
+        cloud.provider: nebula
+        cloud.instance.type: m3.large
```

Comment:
No tenant/project specified, do you assume that the observer can live in whichever project is the default one for the user?

Reply:
Yes, that was the intent.  Only the servers really need to be in the moonshot tenant, everything else can just run in whatever the default tenant might be.